internal/datacoord/garbage_collector.go

```diff
@@ -178,103 +179,146 @@ func (gc *garbageCollector) work() {
 	}
 }
 
+// triggerRecycleFiles is used to trigger recycle files
+func (gc *garbageCollector) triggerRecycleFiles() {
+	select {
+	case gc.fileRecycleCh <- struct{}{}:
```

Comment:
if recyle unused binlog files and index files slow, this will block the above recycle as well.
Let's use a seperate ticker with 24 hour period

Reply:
`fileRecycleCh` is a buffered channel with length 1.
And sent operation is selected with a default branch.
So, it just notifies a file GC after triggering, and it will not block the above meta recycle.
Should I add a new timed(24H) on it? (The stop GC command cannot be applied on file GC).

TODO: A better implementation is executed file GC if and only if meta changed.